service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -148,4 +153,32 @@ void chainCallsTogether() throws Exception {
         newRunCaptor.getValue().submissionTimestamp(),
         greaterThan(OffsetDateTime.now().minus(Duration.ofSeconds(60))));
   }
+
+  @Test
+  void tooManyRecordIds() throws Exception {
+
+    final String workflowUrl = "www.example.com/wdls/helloworld.wdl";
+    final String recordType = "MY_RECORD_TYPE";
+    final String recordIds = "[ \"RECORD1\", \"RECORD2\", \"RECORD3\" ]";
+    final String recordAttribute = "MY_RECORD_ATTRIBUTE";
+    final int recordAttributeValue = 100;
+    RecordAttributes recordAttributes = new RecordAttributes();
+    recordAttributes.put(recordAttribute, recordAttributeValue);
+    final String outputDefinitionAsString =
+        """
+        [ {
+          "output_name" : "myWorkflow.myCall.outputName1",
+          "output_type" : "String",
+          "record_attribute" : "foo_rating"
+        } ]""";
+
+    String request =
+        requestTemplate.formatted(workflowUrl, outputDefinitionAsString, recordType, recordIds);
+
+    MvcResult result =
+        mockMvc
+            .perform(post(API).content(request).contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().is4xxClientError())
+            .andReturn();
```

Comment:
Interesting..... presumably this works because all of the mocks are created outside of the test method? But because all of the "when(x) / do(y)" setup is done in the `chainCallsTogether` test, is this now brittle to running these tests in the opposite order? Or running this test but not the first one?